modules/fedimint-ln-server/src/lib.rs

```diff
@@ -855,16 +840,29 @@ impl Lightning {
         Ok(Lightning { cfg, btc_rpc })
     }
 
-    /// Returns the median block height if enough peers have contributed
-    pub async fn consensus_height(&self, dbtx: &mut ModuleDatabaseTransaction<'_>) -> Option<u64> {
-        let peer_heights = dbtx.get_value(&BlockHeightKey).await.unwrap_or_default();
-        if peer_heights.len() < self.cfg.consensus.threshold() {
-            return None;
+    pub async fn block_height(&self) -> u64 {
```

Comment:
This function seems a bit dangerous, accidental use in consensus logic would break consensus.

Reply:
We could rename it to local_block_height or something?